src/Sylius/Bundle/CoreBundle/Migrations/Version20230426154358.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Sylius\Bundle\CoreBundle\Doctrine\Migrations\AbstractMigration;
+
+final class Version20230426154358 extends AbstractMigration
```

Comment:
Could you merge it into `Version20230426153930`?

Reply:
Both migrations extend different AbstractMigration class, and I'm not quite sure how they can be merged

Reply:
Oh I see, one is for PostgreSQL :+1:

Reply:
![image](https://github.com/Sylius/Sylius/assets/9448101/5088960b-3c71-4983-ab57-6a8f2a6ad689)